templates/search/search.html

```diff
@@ -9,15 +11,14 @@
 {% block content %}
 
 {% include './blocks/search_landing_hero.html' %}
-
-    <div class="search-landing-buckets">
-        <div class="tna-container">
+    {% comment %}<div class="search-landing-buckets">
```

Comment:
**Suggestion:** Could think about explaining why this is being commented out so it does not make it into any real code.